internal/dinosaur/pkg/services/telemetry.go

```diff
@@ -30,19 +53,26 @@ func (t *Telemetry) RegisterTenant(central *dbapi.CentralRequest) {
 			"Region":          central.Region,
 			"Tenant ID":       central.ID,
 		}
-		t.config.Telemeter().Group(central.ID, central.ID, props)
+		t.config.Telemeter().Group(central.ID, user, props)
 	}
```

Comment:
nit: we can save indentation

```suggestion
	if !t.enabled() {
		return
	}
	user, err := getUserFromContext(ctx)
        if err != nil {
	        glog.Warning(errors.Wrap(err, "cannot get telemetry user from context claims"))
	        return
        }
        props := map[string]any{
	        "Cloud Account":   central.CloudAccountID,
	        "Cloud Provider":  central.CloudProvider,
	        "Instance Type":   central.InstanceType,
	        "Organisation ID": central.OrganisationID,
	        "Region":          central.Region,
	        "Tenant ID":       central.ID,
        }
        t.config.Telemeter().Group(central.ID, user, props)
```